src/conversions.jl

```diff
@@ -1225,8 +1240,8 @@ to_spritemarker(marker::AbstractMatrix{<: Colorant}) = marker
 A `Symbol` - Available options can be printed with `available_marker_symbols()`
 """
 function to_spritemarker(marker::Symbol)
-    if haskey(_marker_map, marker)
-        return to_spritemarker(_marker_map[marker])
+    if haskey(default_marker_map(), marker)
```

Comment:
Hm, should we really always re-create the Dict when looking something like this up?
Now, that markers even get created in quite the complex way, this seems pretty expensive, to just be called in the default convert function?
I guess the best way would be to memoize `default_marker_map` ... 
Maybe just with `if isempty(DEFAULT_MARKER_MAP); ... create default markers...; end; return DEFAULT_MARKER_MAP;`

Reply:
ah yeah I think I did this at the time so I could Revise the markers